src/error.rs

```diff
@@ -49,23 +49,23 @@ pub struct FinalError {
 impl Display for FinalError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         // Title
-        writeln!(f, "{}[ERROR]{} {}", *RED, *RESET, self.title)?;
+        write!(f, "{}[ERROR]{} {}", *RED, *RESET, self.title)?;
 
         // Details
         for detail in &self.details {
-            writeln!(f, " {}-{} {}", *WHITE, *YELLOW, detail)?;
+            write!(f, "\n - {}{}{}", *YELLOW, detail, *RESET)?;
```

Comment:
FYI, just in case you don't know it already, `WHITE != RESET` but not in all environments.

Reply:
imo `RESET` is better here for terminals with a light background

Reply:
Ooooooh trueeeeee, agreed, I have never even thought about this before, always be using dark theme.